### PR TITLE
Update RadioNodeList description to reflect this collections content better

### PR DESCRIPTION
**Pull request by vinhill**
### Description

See step 4 in [`namedItem(name)`](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#dom-htmlformcontrolscollection-nameditem). Despite the name, a `RadioNodeList` is a collection of elements with a specified `name` or `id`, but does not necessarily contain radio buttons.

### CODE REVIEW

File: files/en-us/web/api/radionodelist/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ browser-compat: api.RadioNodeList
 
 {{APIRef("HTML DOM")}}
 
-The **`RadioNodeList`** interface represents a collection of radio elements in a {{HTMLElement("form")}} or a {{HTMLElement("fieldset")}} element.
+The **`RadioNodeList`** interface represents a collection of elements in a {{HTMLElement("form")}} or a {{HTMLElement("fieldset")}} element whose `name` or `id` match a specified value.
```
Comment (wbamberg) [inline files/en-us/web/api/radionodelist/index.md:10]:
```suggestion
The **`RadioNodeList`** interface represents a collection of elements in a {{HTMLElement("form")}} or a {{HTMLElement("fieldset")}} element, returned by a call to {{domxref("HTMLFormControlsCollection.namedItem()")}}.
```

To me, reading "match a specified value" makes me think "what specified value?". But AFAIK this object is only returned from `namedItem()`, and this page does not mention `namedItem()` at all.

**Review by wbamberg — COMMENTED** [review]
Thank you @vinhill , and sorry to be slow reviewing this PR.

**Comment by vinhill** [comment]
No problem, the suggested change sounds good.

**Review by wbamberg — APPROVED** [review]
👍 thank you @vinhill !

**Pull request merged**